clio-server/src/main/scala/org/broadinstitute/clio/server/ClioServer.scala

```diff
@@ -71,7 +76,31 @@ object ClioServer
   override val gvcfService =
     new GvcfService(persistenceService, searchService)
 
-  def beginStartup(): Unit = serverService.beginStartup()
+  def run(): Unit = {
+    val startupAttempt = for {
+      _ <- serverService.beginStartup()
+      recoveredUbamCount <- persistenceService.recoverMetadata(
```

Comment:
Note to self: this for-comprehension is causing the gvcf recovery to wait for the ubam recovery, but they could run in parallel.